bin/llvm-kompile-clang

```diff
@@ -149,7 +149,7 @@ fi
 # straight to invoking the C++ compiler.
 if [ "$main" != "python_ast" ]; then
   if [ "$lto" = "lto" ]; then
-    flags+=("-flto" "-Wl,-mllvm,-tailcallopt")
+    flags+=("-flto" "-Wl,-mllvm")
```

Comment:
```suggestion
    flags+=("-flto")
```

The flags `-Wl,-mllvm,-tailcallopt` are all connected and should therefore be removed together. When passed to `clang++`, `-Wl` states "pass everything following a comma to the linker's command line", and similarly when the _linker_ sees `-mllvm` it passes everything following the comma to the LLVM-internal options processing.

The net effect of this nested flag is to pass `-tailcallopt` to the LLVM-internal components that previously dealt with translating `fastcc` calls to `tailcc`.